src/map/skill.cpp

```diff
@@ -13487,15 +13478,9 @@ TIMER_FUNC(skill_castend_id){
 		//Consume SP/spheres
 		skill_consume_requirement(sd,ud->skill_id, ud->skill_lv,1);
 		status_set_sp(src, 0, 0);
-		sc = &sd->sc;
```

Comment:
Reviewing on mobile, but you erase this here. Is it needed later in the function? Should we remove the `sc` variable completely?

Reply:
It's also a mystery to me but sc is defined at the top of the function and used also just 5 lines above:

`status_change *sc = status_get_sc(src);`

`!(sc && sc->getSCE(SC_FOGWALL))`

I don't understand why it was overwritten with &sd->sc here. Doesn't that return the exactly same sc object?

Reply:
I looked over the file on the PC and can agree that the removal here is safe. The `sc` assignment here is a duplicate, most likely from a left over in a refactor.